numjuggler/main.py

```diff
@@ -260,7 +269,7 @@ def main():
             # as well.
 
             import re
-            r = re.compile('(u)(\d+)')
+            r = re.compile(r'(u)(\d+)')
```

Comment:
Why this change? Both variants (with ``r`` and without) create the same pattern both in Python2 and Python3.

Reply:
There was Python 3.7 warning: the pattern construct without 'r' became deprecated. Just to stop these warnings.

Reply:
Ok, thanks.